clouduniqueid/clouds/aws/__init__.py

```diff
@@ -84,48 +88,48 @@ def get_unique_id(
                 errorMsg = errorMsg[:-1]
 
                 logger.error(f"AWS{errorMsg} keys required in data parameter")
-                raise ValueError(
-                    f"Invalid parameters provided, AWS{errorMsg} keys required in data parameter,\
-                        uniqueId format for resource {service} {resourceType} is {uniqueIdFormat}",
-                )
+                # raise ValueError(
+                #     f"Invalid parameters provided, AWS{errorMsg} keys required in data parameter,\
+                #         uniqueId format for resource {service} {resourceType} is {uniqueIdFormat}",
+                # )
+                return ""
 
             else:
                 if uniqueIds[service][resourceType].startswith("hashlib"):
                     if resourceType == "acl":
-                        try:
-                            data = "{}:{}:{}:{}:{}:{}:{}:{}".format(
-                                accountId,
-                                data["owner"],
-                                data["ownerid"],
-                                data["type"],
-                                data["displayname"],
-                                data["granteeid"],
-                                data["uri"],
-                                data["permission"],
-                            )
-                        except Exception as e:
-                            raise ValueError(
-                                f"Invalid parameters provided, owner, ownerid, type, displayname, granteeid, uri, permission keys required in data parameter: {e}",
-                            )
+                        return "{}:{}:{}:{}:{}:{}:{}:{}".format(
+                            accountId,
+                            data.get("owner"),
+                            data.get("ownerid"),
+                            data.get("type"),
+                            data.get("displayname"),
+                            data.get("granteeid"),
+                            data.get("uri"),
+                            data.get("permission"),
+                        )
+
                     return eval(
                         eval(f"f'{uniqueIds[service][resourceType]}'").replace(" ", ""),
                     )
+
                 return eval(f"f'{uniqueIds[service][resourceType]}'").replace(" ", "")
 
     def get_unique_id_format(self, service: str, resourceType: str) -> str:
```

Comment:
<b>Vulnerability:</b> Detected the use of eval(). eval() can be dangerous if used to evaluate dynamic content. If this content can be input from outside the program, this may be a code injection vulnerability. Ensure evaluated content is not definable by external sources.<br/>
                            <b>Lines</b>: <b>Lines</b>: 115 to 115<br/>
                            <b>OWASP:</b> A03:2021 - Injection<br/>
                            <b><a href="https://console.cloudanix.com/code?t=vulnerabilities" target='_blank'>Click here for more details</a></b>